src/app/pages/ConsensusDashboardPage/RuntimePreview.tsx

```diff
@@ -128,7 +147,11 @@ const RuntimePreview: FC<RuntimePreviewProps> = ({ prominentItem, network, runti
           )}
         </Typography>
       </div>
-      <StyledList>
+      <dl
+        className="m-0 mb-4 grid grid-cols-[minmax(80px,1fr)_1fr] gap-y-1 
+               [&_dt]:flex [&_dd]:flex [&_dt]:items-start [&_dd]:justify-end 
```

Comment:
please move tailwind classes to target elements. instead of `[&_dt]:flex [&_dd]:flex [&_dt]:items-start [&_dd]:` etc
just put classes when you need them

```
      <dl className="m-0 mb-4 grid grid-cols-[minmax(80px,1fr)_1fr] gap-y-1">
        <dt className="flex items-start py-1">
        <dd className="flex justify-end py-1">
```